aws/resource_aws_guardduty_member.go

```diff
@@ -37,6 +39,23 @@ func resourceAwsGuardDutyMember() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"relationship_status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"invite": {
```

Comment:
If we try to properly `d.Set("invite", ...)` below, we should add `Computed: true` to this attribute for now 👍  At some point a resource update function could be added to support inviting an existing member that hasn't been invited yet.